src/StatusQ/Core/StatusChatListCategoryItemTooltipSettings.qml

```diff
@@ -0,0 +1,7 @@
+import QtQuick 2.14
+
+QtObject {
+    property StatusTooltipSettings add: StatusTooltipSettings {}
+    property StatusTooltipSettings menu: StatusTooltipSettings {}
+    property StatusTooltipSettings toggle: StatusTooltipSettings {}
```

Comment:
These will live on `StatusChatListAndCategories`, so we should probably name them:

```
StatusChatListCategories {
  categoryAddButton.tooltip: ...
  categoryMenuButton.tooltip: ..
}
```